src/makeRelativeDate.ts

```diff
@@ -0,0 +1,164 @@
+type RelativeDateOptions = {
+	notAfter?: number;
+	format?: FormatType;
+	showTime?: boolean;
+};
+
+type FormatType = 'short' | 'med' | 'long';
+type PeriodType = 's' | 'm' | 'h' | 'd';
+
+const dayOfWeek = (day: number): string =>
+	[
+		'Sunday',
+		'Monday',
+		'Tuesday',
+		'Wednesday',
+		'Thursday',
+		'Friday',
+		'Saturday',
+	][day];
+
+const monthAbbr = (month: number): string =>
```

Comment:
I think month names are more commonly described as 'short' or 'long'

```suggestion
const shortMonth = (month: number): string =>
```

Reply:
Fixed in a separate commit 👍